src/lib.rs

```diff
@@ -414,6 +429,52 @@ where
     runner.run(f);
 }
 
+/// Run the given function using random scheduler according to a seed.
+///
+/// This function allows deterministic replay of a failing schedule, as long as `f` contains no
+/// uncontrolled non-determinism. Uncontrolled non-determinism can be checked using
+/// `check_uncontrolled_nondeterminism`.
+///
+/// This is a convenience function for constructing a [`Runner`] that uses
+/// [`RandomScheduler::new_from_seed`](scheduler::RandomScheduler::new_from_seed) for one
+/// iteration.
+pub fn replay_random_from_seed<F>(f: F, seed: u64)
```

Comment:
Not sure there is any point in having this function. This is just `check_random_with_seed(f, seed, 1)`.